src/main/java/DeleteElementsFromArrayList2.java

```diff
@@ -0,0 +1,36 @@
+
+import java.util.Scanner;
+import java.util.ArrayList;
+//Objects are compared for equality using their equals() method
+//Override the equals method in your own class
+ class MyPerson {
+
+    String name;
+    MyPerson(String name) {this.name=name;}
+
+    //@override
```

Comment:
read what is @override.
@override is correct.

Reply:
what is the use of @override.

Reply:
@Override @Override annotation informs the compiler that the element is meant to override an element declared in a superclass. ... While it is not required to use this annotation when overriding a method, it helps to prevent errors.

Using @Override annotation while overriding a method is considered as a best practice for coding in java because of the following two advantages: 1) If programmer makes any mistake such as wrong method name, wrong parameter types while overriding, you would get a compile time error.

Reply:
Java - Overriding. ... The benefit of overriding is: ability to define a behavior that's specific to the subclass type, which means a subclass can implement a parent class method based on its requirement. In object-oriented terms, overriding means to override the functionality of an existing method.

Reply:
why can not you remove comment it

Reply:
remove the comment //@override and see if t works.
And create the person calss in another file. 
like Person.Java so it will be easy to read.